vector-im/element-android
# [Device management] Rename a session (PSG-747)

## Type of change


## Content


Adding a new screen to rename a given session. It can be reached from the session overview screen using the overflow menu.

## Motivation and context


Closes #7158 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/191469540-9389f1d2-72da-4db4-b122-540512a35576.gif" width=300 />

## Tests



- Enable the new device manager feature flag
- Go to Settings -> Security & Privacy -> Show all sessions (V2, WIP)
- Press a session entry to go to a session overview
- Press the overflow menu
- Press the rename action
- Check the focus is on the text field and the keaboard is opened when landing on the screen
- Edit the name
- Press the "Save" button
- Check the session name is correctly updated everywhere
- Press a session entry to go to a session overview
- Press the overflow menu
- Press the rename action
- Edit the name
- Press the close button
- Check the session name has not been changed

## Tested devices

- OS version(s): Android 11

## Checklist
